src/node.js

```diff
@@ -10,187 +10,218 @@ const shutdown = require('shutdown')
 const path = require('path')
 const join = path.join
 
-const rootPath = process.env.testpath ? process.env.testpath : __dirname
-
-const ipfsDefaultPath = findIpfsExecutable()
+const BinWrapper = require('bin-wrapper')
+
+// TODO maybe this should go to some config file
+const DIST_BASE = 'https://dist.ipfs.io/go-ipfs'
+const VERSION = 'v0.4.1'
+const PROJECT_BASE = DIST_BASE + '/' + VERSION + '/go-ipfs_' + VERSION + '_'
+const IPFS_DEFAULT_EXEC_PATH = path.join(__dirname, '..', 'vendor')
+
+const bin = new BinWrapper()
+    .src(PROJECT_BASE + 'darwin-386.tar.gz', 'darwin', 'ia32')
+    .src(PROJECT_BASE + 'darwin-amd64.tar.gz', 'darwin', 'x64')
+    .src(PROJECT_BASE + 'freebsd-amd64.tar.gz', 'freebsd', 'x64')
+    .src(PROJECT_BASE + 'linux-386.tar.gz', 'linux', 'ia32')
+    .src(PROJECT_BASE + 'linux-amd64.tar.gz', 'linux', 'x64')
+    .src(PROJECT_BASE + 'linux-arm.tar.gz', 'linux', 'arm')
+    .src(PROJECT_BASE + 'windows-386.tar.gz', 'win32', 'ia32')
+    .src(PROJECT_BASE + 'windows-amd64.tar.gz', 'win32', 'x64')
+    .use(process.platform === 'win32' ? 'ipfs.exe' : 'ipfs')
+    // .version('>=1.71');
```

Comment:
This should probably be removed. `bin-wrapper` has support to check the binary version against what you're expecting, but unfortunately it doesn't allow us to set the arguments used to run the version command and it is currently defaulting to `--version`.